.github/workflows/ci.yml

```diff
@@ -85,20 +75,24 @@ jobs:
         with:
           node-version: 14
 
-      - name: Set dependencies cache
-        uses: actions/cache@v2
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2.0.1
         with:
-          path: ~/.cache/yarn
-          key: lint-dependencies-${{ hashFiles('yarn.lock') }}
-          restore-keys: |
-            lint-dependencies-${{ hashFiles('yarn.lock') }}
-            lint-dependencies-
+          version: 6.15.1
+
+      - name: Set node version to ${{ matrix.node_version }}
```

Comment:
lint job has no matrix and setup-node is used twice (here and line 73ff)

The one in line 73ff can be deleted and  ${{ matrix.node_version }} hardcoded to 14

alternatively a 1x1 matrix with just ubuntu 14 could be used so that it's more similar to the build job